src/components/flex/Flex.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { ReactNode } from 'react';
+import './flex.css';
+
+interface IFlexProps {
+  className?: string;
+  justify?:
+    | 'flex-start'
+    | 'flex-end'
+    | 'center'
+    | 'space-between'
+    | 'space-around'
+    | 'space-evenly'
+    | 'start'
+    | 'end'
+    | 'left';
+  align?:
+    | 'stretch'
```

Comment:
Estos puedes hacer un type y exportarlo 

`export type alingTypes = 'flex-start' | 'flex-end';`

y llamarlo 

 align?: alingTypes,
 
 lo mismo con el de arriba

Reply:
Fixing it now